jobs/web/spec

```diff
@@ -1232,6 +1232,12 @@ properties:
       Port of the Datadog Agent's dogstatsd server to emit events to.
     default: 8125
 
+    datadog.agent_uds_filepath:
+    env: CONCOURSE_DATADOG_AGENT_UDS_FILEPATH
```

Comment:
This indentation looks a little busted. Could you backport https://github.com/concourse/concourse-bosh-release/pull/158/commits/a7f307ccb83e1e6d8ab1915725bb61802a4fddce instead